crates/timber/src/formatter.rs

```diff
@@ -4,23 +4,29 @@ use tracing_subscriber::fmt;
 
 use std::io;
 
-pub(crate) fn least_verbose(level: Level) {
-    let format = fmt::format().without_time().with_target(false).compact();
+pub(crate) fn pretty_print(level: Level) {
+    let format = fmt::format()
+        .without_time()
+        .with_target(false)
+        .pretty()
+        .with_source_location(false);
```

Comment:
If we (the rover team) are debugging-debugging, I think the `source_location` is pretty useful. Let's keep it!